services/evaluator/tests/integration/test_real_llm_providers.py

```
@@ -0,0 +1,474 @@
+
+Candidate: Yes, what's the team's approach to code reviews and what development methodologies do you follow?
+            """.strip()
+        )
+    
+    @pytest.mark.real_api
+    @pytest.mark.asyncio
+    async def test_openai_gpt5_real_call(self, settings, sample_interview):
+        """Test real OpenAI GPT-5 API call"""
+    @pytest.mark.real_api
+    @pytest.mark.asyncio
+    async def test_openai_gpt5_real_call(self, settings, sample_interview):
```

Ruff (0.13.3)

Redefinition of unused `test_openai_gpt5_real_call` from line 80

(F811)